yaksh/python_assertion_evaluator.py

```diff
@@ -47,11 +68,13 @@ def check_code(self, user_answer, file_paths, test_case):
             info = traceback.extract_tb(tb)
             fname, lineno, func, text = info[-1]
             text = str(test_case).splitlines()[lineno-1]
-            err = "{0} {1} in: {2}".format(type.__name__, str(value), text)
+            err = ("-----\nExpected Test Case:\n{0}\n"
```

Comment:
Looks ugly, can't it be some HTML code or can this be delegated to something else?